src/Valuestore.php

```diff
@@ -48,12 +47,12 @@ protected function setFileName(string $fileName)
     /**
      * Put a value in the store.
      *
-     * @param string|array    $name
-     * @param string|int|null $value
+     * @param array|string                $name
```

Comment:
Can't we just type this instead of the docblock?

Reply:
I've added the type-hint to the parameter.